Popup/Popup.js

```diff
@@ -56,6 +56,16 @@ const PopupBase = kind({
 		noAnimation: PropTypes.bool,
 		onClose: PropTypes.func,
 		onHide: PropTypes.func,
+
+		/**
+		 * Puts the popup on top of screen.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		onScreenDisplay: PropTypes.bool,
```

Comment:
`onScreenDisplay` name is not straightforward for me. 
What about to have `position` like
```
position: PropTypes.oneOf(['bottom', 'center', 'fullscreen', 'left', 'right', 'top']),
```
for more flexibility?
As of now, we can probably have only `center` and `top` but we can support with the same prop when the new requirement comes like `bottom` `right` left` and even `fullscreen`
@ybsung FYI, I'd like to have the same prop as Popup in Sandstone

Reply:
ok, i renamed it to "position". Default is center and "top" is also handled